### PR TITLE
Updated discord links closes  #2274

### LINKED ISSUE #2274: [OTHER] Update Discord links
### What would you like to share?

We're using a new link for The Algorithms' Discord invitation.
All Discord invitation links should point to `https://the-algorithms.com/discord/`.

### Additional information

This is a good first issue. Please leave it open to first-time contributors and beginners. 🙂

### PR DESCRIPTION
**Pull request by zafar-hussain**
#### Checklist

- [x] Added description of change
- [ ] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#New-File-Name-guidelines)
- [ ] Added tests and example, test must pass
- [ ] Added documentation so that the program is self-explanatory and educational - [Doxygen guidelines](https://www.doxygen.nl/manual/docblocks.html)
- [x] Relevant documentation/comments is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes: 

with reference to issue  #2274 "[OTHER] Update Discord links", four discord links were updated to https://the-algorithms.com/discord/

### CODE REVIEW

**Comment by zafar-hussain** [comment]
@reviewers

please, if you can add "hacktoberfest-accepted" label, if this PR is fulfilled

cheers

**Comment by zafar-hussain** [comment]
@Panquesito7

Please if you can check and advice on this PR

cheers

File: strings/naive_pattern_search.cpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
+// # Part of Cosmos by OpenGenus Foundation
```
Comment (Panquesito7) [inline strings/naive_pattern_search.cpp:1]: Please make a separate PR for this change. Thank you. 🙂

File: strings/test_naive_pattern_search.cpp [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/**
```
Comment (Panquesito7) [inline strings/test_naive_pattern_search.cpp:1]: Same here.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**